tests/rkt_volume_test.go

```diff
@@ -0,0 +1,170 @@
+	output, err := exec.Command("../bin/actool",
+		"patch-manifest",
+		"--overwrite",
+		"--exec=/inspect --read-file",
+		"rkt-inspect.aci",
+		"rkt-inspect-read-file.aci").Output()
+	if err != nil {
+		panic(fmt.Sprintf("Cannot create ACI: %v: %v\n", err, output))
+	}
+	defer os.Remove("rkt-inspect-read-file.aci")
+
+	output, err = exec.Command("../bin/actool",
+		"patch-manifest",
+		"--overwrite",
+		"--exec=/inspect --write-file --read-file",
+		"rkt-inspect.aci",
+		"rkt-inspect-write-file.aci").Output()
+	if err != nil {
+		panic(fmt.Sprintf("Cannot create ACI: %v: %v\n", err, output))
+	}
+	defer os.Remove("rkt-inspect-write-file.aci")
+
+	output, err = exec.Command("../bin/actool",
+		"patch-manifest",
+		"--overwrite",
+		"--exec=/inspect --read-file",
+		"--mounts=dir1,path=/dir1,readOnly=false",
+		"rkt-inspect.aci",
+		"rkt-inspect-vol-rw-read-file.aci").Output()
+	if err != nil {
+		panic(fmt.Sprintf("Cannot create ACI: %v: %v\n", err, output))
+	}
+	defer os.Remove("rkt-inspect-vol-rw-read-file.aci")
+
+	output, err = exec.Command("../bin/actool",
+		"patch-manifest",
+		"--overwrite",
+		"--exec=/inspect --write-file --read-file",
+		"--mounts=dir1,path=/dir1,readOnly=false",
+		"rkt-inspect.aci",
+		"rkt-inspect-vol-rw-write-file.aci").Output()
+	if err != nil {
+		panic(fmt.Sprintf("Cannot create ACI: %v: %v\n", err, output))
+	}
+	defer os.Remove("rkt-inspect-vol-rw-write-file.aci")
+
+	output, err = exec.Command("../bin/actool",
+		"patch-manifest",
+		"--overwrite",
+		"--exec=/inspect --read-file",
+		"--mounts=dir1,path=/dir1,readOnly=true",
+		"rkt-inspect.aci",
+		"rkt-inspect-vol-ro-read-file.aci").Output()
+	if err != nil {
+		panic(fmt.Sprintf("Cannot create ACI: %v: %v\n", err, output))
+	}
+	defer os.Remove("rkt-inspect-vol-ro-read-file.aci")
+
+	output, err = exec.Command("../bin/actool",
+		"patch-manifest",
+		"--overwrite",
+		"--exec=/inspect --write-file --read-file",
+		"--mounts=dir1,path=/dir1,readOnly=true",
+		"rkt-inspect.aci",
+		"rkt-inspect-vol-ro-write-file.aci").Output()
+	if err != nil {
+		panic(fmt.Sprintf("Cannot create ACI: %v: %v\n", err, output))
+	}
+	defer os.Remove("rkt-inspect-vol-ro-write-file.aci")
+
+	tmpdir, err := ioutil.TempDir("", "rkt-tests.")
+	if err != nil {
+		t.Fatalf("Cannot create temporary directory: %v", err)
+	}
+	defer os.Remove(tmpdir)
+
+	tmpfile := filepath.Join(tmpdir, "file")
+	err = ioutil.WriteFile(tmpfile, []byte("host"), 0600)
+	if err != nil {
+		t.Fatalf("Cannot create temporary file: %v", err)
+	}
+	defer os.Remove(tmpfile)
+
+	for i, tt := range volTests {
+		cmd := strings.Replace(tt.rktCmd, "$TMPDIR", tmpdir, -1)
+
+		t.Logf("Running test #%v: %v", i, cmd)
+
+		child, err := gexpect.Spawn(cmd)
+		if err != nil {
+			t.Fatalf("Cannot exec rkt #%v: %v", i, err)
+		}
+
+		err = child.Expect(tt.expect)
+		if err != nil {
+			fmt.Printf("Command: %s\n", cmd)
+			t.Fatalf("Expected %q but not found #%v", tt.expect, i)
+		}
+
+		child.Close()
```

Comment:
Consider using `child.Cmd.Wait()` instead. `Close()` simply kills it, which might have some unwanted effects.